tests/server.https.test.ts

```diff
@@ -0,0 +1,114 @@
+// tests/server.https.test.ts
+// Tests for the serve with HTTPS.
+
+import { afterEach, describe, test, expect, vi } from 'vitest';
+import { extend as createFetch } from 'got';
+
+import { loadConfiguration } from '../source/utilities/config.js';
+import { startServer } from '../source/utilities/server.js';
+
+// The path to the fixtures for this test file.
+const fixture = 'tests/__fixtures__/server/';
+// The configuration from the fixture.
+const config = await loadConfiguration(process.cwd(), fixture, {});
+// A `fetch` instance to make requests to the server.
+const fetch = createFetch({
+  https: { rejectUnauthorized: false },
+});
+
+afterEach(() => {
+  vi.restoreAllMocks();
+});
+
+describe('utilities/server', () => {
+  test('start server with SSL when PFX certificate is supplied', async () => {
+    const address = await startServer({ port: 5001 }, config, {
+      ['--ssl-cert']: 'tests/__fixtures__/server/cert.pfx',
```

Comment:
You could use ``${fixtures}/cert.pfx`` here instead of `tests/..../cert.pfx`.

Reply:
Done